openmetadata-ui/src/main/resources/ui/src/constants/Appearance.constants.ts

```diff
@@ -13,7 +13,7 @@
 export const DEFAULT_THEME = {
   primaryColor: '#1570ef',
   infoColor: '#84caff',
-  successColor: '#6ce9a6',
-  warningColor: '#fec84b',
+  successColor: '#039855',
+  warningColor: '#DC6803',
```

Comment:
Changing this cause an impact to every place did we check or validate all the places?

Reply:
yes, have verified with the application log and the entity version, where we are using this color shade. And it is lot more better now.
Previous : 
<img width="804" alt="image" src="https://github.com/user-attachments/assets/de2a3337-052a-4312-bbc4-e32e39383959" />
Current :
<img width="804" alt="image" src="https://github.com/user-attachments/assets/b66e077a-5e61-4386-aa7b-f4b0a2566162" />